### PR TITLE
Make the containerrick great again

### PR DESCRIPTION
**Pull request by dgonzalezruiz**
The gonsalito did it again! This PR features:

- Improvements given by shellcheck to the scripts
- Improvement of the overall structure of the centerGraphics script
- Improvement of the repo file structure
- Use of different exit codes for different problems

### CODE REVIEW

**Comment by chipironcin** [comment]
You mean
> Overengineering the overengineered containerick

😄 Will review your PR in a bit

File: centerGraphic.sh [hunk 1]
```diff
@@ -1,11 +1,23 @@
 #!/bin/sh
 set -euo pipefail
 
-DEFAULT_COLS=$(stty size 2>/dev/null | awk {'print $2'} || echo "1")
+LANG="C.UTF-8"
```
Comment (chipironcin) [inline centerGraphic.sh]: I would like to maintain this line next to the MAX_LENGTH line because they are closely related

**Comment by chipironcin** [comment]
Rest of it looks good to me.
Thanks for the improvements!!

File: centerGraphic.sh (hunk 1 above)
Comment (dbarranco) [reply centerGraphic.sh]: As an expert in dburrito's PR I would like to give @dgonsudo my 10s.


ddrunk did it again. This is great.

**Comment by chipironcin** [comment]
Thanks @dgonzalezruiz !!

**Comment by dgonzalezruiz** [comment]
The gonsalito is always glad to help

**Pull request merged**